distributions/Dist_yz2668.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np 
+
+class Dist_yz2668(BaseDistribution):
+	def __init__(self):
+		self.f_max = 5
+		self.x_min = 0
+		self.x_max = 1
+
+	def pdf(self, x):
+		"""This is your PDF""" 
+		return (x**2)+3*x+2
+        
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 0.576
```

Comment:
better would be to have an equation like 
`return self.x_max**3 - self.x_min**3`